etl/steps/data/garden/iea/2024-07-04/critical_minerals.meta.yml

```diff
@@ -0,0 +1,34 @@
+definitions:
+  common:
+    processing_level: major
```

Comment:
I think you probably want to add a couple of notes on `description_processing` given what you've said in your notes at the top of the garden step?